demo/index.html

```diff
@@ -1,54 +1,54 @@
 <!doctype html>
 <!--
-  @license
-  Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
-  This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-  The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-  The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-  Code distributed by Google as part of the polymer project is also
-  subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 <html lang="en">
 
-<head>
-  <meta charset="UTF-8">
-  <title>Paper Spinner</title>
-  <script src="../../webcomponentsjs/webcomponents.js"></script>
-  <link rel="import" href="../paper-spinner.html">
-  <style>
+  <head>
+    <meta charset="UTF-8">
+    <title>paper-spinner demo</title>
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <link rel="import" href="../paper-spinner.html">
+    <style>
 
-    paper-spinner.blue .circle { border-color: #4285f4; }
-    paper-spinner.blue::shadow .circle { border-color: #4285f4; }
+      paper-spinner.blue .circle { border-color: #4285f4; }
+      paper-spinner.blue::shadow .circle { border-color: #4285f4; }
```

Comment:
Yeah! I will make a much-gianter PR where I'll clean-up the actual div names in the `paper-spinner` itself, and then the custom properties will make sense. Thanks for suggesting it! :raised_hands: